test/kafka-rtr/mzcompose.py

```diff
@@ -59,8 +59,10 @@ def workflow_simple(c: Composition) -> None:
     )
 
 
-# TODO: All failure modes fail with: timed out before ingesting the source's visible frontier when real-time-recency query issued. input_1 failed to ingest data up to the real-time recency point
-def workflow_resumption(c: Composition) -> None:
+# It is impossible for this test to succeed because the network failures
```

Comment:
More accurate to say that this can't pass without making the RTR-related connections robust to network failure?
Is this something that would only make sense to re-visit in the context of the work to add HA/"always on" behavior? Or resuming these broken connections could add value even before HA work is done?
If this is correct, should we open a issue to work on this at some point, and add it to the mega tracker?

Reply:
For this to work, we would have to do something other than blithely partition `envd` away from the external system. If you partition `envd` away from the external system, you cannot reach out to it to determine its frontier. For this test to work, we would need to do something trickier, like only partition us away from the external system _after_ the thread performing the RTR determination received a response from the query that determines the external frontier.

Reply:
@MaterializeInc/testing I'm going to remove these functions because as-is they cannot succeed but will leave a TODO for @def- to investigate a way to make them work with a link to the commits in https://github.com/MaterializeInc/materialize/pull/25566 that have the original tests.

Reply:
Fine with me.